docs/sphinx/_toc.yml.in

```diff
@@ -37,21 +37,48 @@ subtrees:
 
 - caption: Reference
   entries:
-  - file: doxygen/html/index
+  - file: reference/hip_runtime_api_reference
+    subtrees:
+    - entries:
```

Comment:
I would add the Modules sub-topic here, and then have the list of modules as sub-topics.

Reply:
Updated.